testsuite/tests/libratest/throughput_test.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use benchmark::{
+    bin_utils::{create_ac_clients, measure_throughput},
+    ruben_opt::parse_swarm_config_from_dir,
+    txn_generator::gen_accounts,
+    Benchmarker,
+};
+use libra_swarm::swarm::LibraSwarm;
+use libra_wallet::wallet_library::WalletLibrary;
+use num::traits::Float;
+use rusty_fork::{rusty_fork_id, rusty_fork_test, rusty_fork_test_name};
+use statistical::{mean, population_standard_deviation};
+use std::fmt::Display;
+
+fn calculate_avg_std<T: Float + Display>(sequence: &[T], name: &str) {
+    // Output json-formatted result.
+    println!(
+        "{{\"{}_mean\": {:.2}, \"{}_stdev\": {:.2}}}",
+        name,
+        mean(sequence),
+        name,
+        population_standard_deviation(sequence, None),
+    );
+}
+
+rusty_fork_test! {
+    #[test]
+    fn test_throughput() {
+        let (num_nodes, num_accounts, num_clients) = (4, 32, 4);
+        let (num_rounds, num_epochs, stagger_ms) = (2, 4, 1);
```

Comment:
curious about how these parameters are chosen? especially for `num_accounts`, `num_rounds`, `num_epochs`. we might want to tune these parameters to make the test result relatively stable and at the same efficient.

Reply:
For now the parameters are chosen by the criteria that anyone can run/test locally, but still generating sufficient large number of TXNs (2k TXNs per epoch in this setting).
Soon we will update this test to send these TXNs at a speed that we expect CI's machine is able to achieve, say 200 Tps, and in the test we can warn if the throughput doesn't fall within 80%-100% of the request speed.